knimin/lib/tests/test_format.py

```diff
@@ -34,6 +35,26 @@ def test_format_epmotion_file(self):
         obs = obs.splitlines()
         self.assertEqual(obs[2], '1,a2,1,5,1.200,1')
 
+    def test_format_normalization_echo_pick_list(self):
+        vol_sample = np.zeros((4, 4), dtype=np.float)
```

Comment:
If these two lines are to initiate an array of 1.5's, why not `np.full((4, 4), 1.5, dtype=np.float)`.